### PR TITLE
Auth page: Increased the space between the fields.

### PR DESCRIPTION
**Pull request by Visham32**
Used the br tag.

### CODE REVIEW

File: app/views/auth.html [hunk 1]
```diff
@@ -12,8 +12,9 @@
 
     <div class="loginFields">
       <input type="text" ng-model="user.email" class="form-control" placeholder="Email Address">
-
+      <br>
```
Comment (jaylenw) [inline app/views/auth.html]: Adding the break tags makes it look better, however I would prefer to use css to add spacing for the input fields. Remove the br tags and check the stackoverflow links.

Check this link at stackoverflow http://stackoverflow.com/questions/16379187/spacing-between-form-fields and http://stackoverflow.com/questions/1896567/how-can-i-add-spaces-between-two-input-lines-using-css

Lets try adding this to the css file. Lets see if 10px will look correctly.
input[type="text"] {margin: 0 0 10px 0;}
input[type="password"] {margin: 0 0 10px 0;}
If 10px is too large we can decrease it. Then everything will be all set! Once done, just commit and push your changes back up and I will merge it. 

Thanks!

File: app/views/auth.html [hunk 2]
```diff
@@ -12,8 +12,9 @@
...
       <input type="password" ng-model="user.password" class="form-control" placeholder="Password">
+      <br>
```
Comment (jaylenw) [inline app/views/auth.html]: remove

File: app/views/auth.html (hunk 1 above)
Comment (jaylenw) [reply app/views/auth.html]: We also have to make sure it looks good when the user wants to log in as well for registering.

**Review by jaylenw — COMMENTED** [review]
Remove the <br> tag and add css to make proper spacing to our desire.

**Comment by Visham32** [comment]
Ok. Thank you for the links.

File: app/views/auth.html [hunk 3]
```diff
@@ -11,11 +11,17 @@
...
 
     <div class="loginFields">
+<label>
       <input type="text" ng-model="user.email" class="form-control" placeholder="Email Address">
-
-      <input type="password" ng-model="user.password" class="form-control" placeholder="Password">
-      <input type="password" ng-model="user.confirmpassword" ng-show="amRegistering"  class="form-control" ng-class="{fieldInvalid: (user.password != user.confirmpassword)}" placeholder="Confirm Password">
-  </div>
+</label>
+<label>
```
Comment (jaylenw) [inline app/views/auth.html:17]: Make sure you be mindful of the spacing/tabs for the tags.

Following the indentation of the code makes it easier to read.

**Review by jaylenw — COMMENTED** [review]
Thanks for adding the feature in spacing out the input fields. It looks so much better.

I will fix the indentation and ng-class. 

I will be merging this in shortly. Thanks again!

File: app/views/auth.html [hunk 4]
```diff
@@ -11,11 +11,17 @@
...
-      <input type="password" ng-model="user.password" class="form-control" placeholder="Password">
-      <input type="password" ng-model="user.confirmpassword" ng-show="amRegistering"  class="form-control" ng-class="{fieldInvalid: (user.password != user.confirmpassword)}" placeholder="Confirm Password">
-  </div>
+</label>
+<label>  
+<input type="password" ng-model="user.password" class="form-control" placeholder="Password">
+</label>
+<label>      
+<input type="password" ng-model="user.confirmpassword" ng-show="amRegistering"  class="form-control" ng-
+class="{fieldInvalid: (user.password != user.confirmpassword)}" placeholder="Confirm Password">
```
Comment (jaylenw) [inline app/views/auth.html:22]: this is where ng-class is broken. A new line was created when it should be one single line.

**Pull request merged**